allocations/views.py

```diff
@@ -49,6 +50,8 @@ def view( request ):
         tas = TASClient()
         resp = tas.projects_for_group('Chameleon')
         for p in resp:
+            mapper = ProjectAllocationMapper(request)
+            p = mapper.map(p, get_redis_data = False)
```

Comment:
Nitpick: after seeing this in a few places, I'm thinking that `fetch_balance` is probably a better name for this argument. The fact that it's Redis shouldn't really matter (and in fact, we are getting it from a balance service anyways, not Redis directly).